watertap/core/util/scaling.py

```diff
@@ -24,20 +24,30 @@ def transform_property_constraints(self):
         var_str = p.name
         if p.method is not None and self.is_property_constructed(var_str):
             var = getattr(self, var_str)
+            msg = (
```

Comment:
@MarcusHolly I modified your utility function to not show logger warnings for in cases where a variable is fixed.